src/libraries/System.Diagnostics.EventLog/tests/EventLogTests/EventLogSourceCreationTests.cs

```diff
@@ -11,6 +11,7 @@ public class EventLogSourceCreationTests
     {
         [Trait(XunitConstants.Category, "EventLog")] // Unreliable Win32 API call
```

Comment:
@Anipik was the intention for this trait to be ignored on CI and regular runs? If so we should change it to "IgnoreForCI"

Reply:
That was my fault: https://github.com/dotnet/runtime/commit/48a222cbbbee6c39d622a9daee86f8f185263d5f 😢.

Reply:
oh, haha. I'll update the PR 😄

Reply:
Fixed.